### PR TITLE
feat: change uid

### LINKED ISSUE #14: ImplementationClassUID invalid
The **ImplementationClassUID (0002, 0012)** tag is set to an invalid value.

In [this line]( https://github.com/qurit/rt-utils/blob/20a27c48530a6c459cdeb7a869865ce7c770c677/rt_utils/utils.py#L31) it is set to _PYDICOM_ROOT_UID_ value defined in pydicom. 
This value is just the root of the pydicom's implementation UID and not the full UID("1.2.826.0.1.3680043.8.498."). The full implementation class UID of pydicom is defined in _PYDICOM_IMPLEMENTATION_UID_ ( "1.2.826.0.1.3680043.8.498.1")

Found out that it is invalid while running the DICOM validation tool [dciovfy](http://dclunie.com/dicom3tools/dciodvfy.html).

### PR DESCRIPTION
**Pull request by asim-shrestha**
fixes #14

### CODE REVIEW

**Comment by asim-shrestha** [comment]
Alternatively, could add a different number than 1 to the root uid to get something more unique.

`PYDICOM_IMPLEMENTATION_UID` is just `PYDICOM_ROOT_UID` with a 1 added
image

**Comment by awtkns** [comment]
@carluri Should we be using Pydicom's implementation UID or should we be making a new Implementation UID for rt-utils

File: rt_utils/utils.py [hunk 1]
```diff
@@ -28,7 +28,7 @@
 
 
 class SOPClassUID:
-    RTSTRUCT_IMPLEMENTATION_CLASS = PYDICOM_ROOT_UID  # TODO find out if this is ok
+    RTSTRUCT_IMPLEMENTATION_CLASS = PYDICOM_IMPLEMENTATION_UID  # TODO find out if this is ok
     CT_IMAGE_STORAGE = '1.2.840.10008.5.1.4.1.1.2'
```
Comment (awtkns) [inline rt_utils/utils.py:32]: Do we still need this?
```suggestion
```

Comment (asim-shrestha) [reply rt_utils/utils.py:32]: doesn't look like its used anywhere

**Comment by awtkns** [comment]
@asim-shrestha Good to merge?

**Pull request merged**